coldfront/core/project/views_/renewal_views/request_views.py

```diff
@@ -128,7 +129,7 @@ def __init__(self, *args, **kwargs):
 
     @staticmethod
     def create_allocation_renewal_request(requester, pi, computing_allowance,
-                                          allocation_period, pre_project,
+                                          allocation_period, renewal_survey_answers, pre_project,
```

Comment:
Minor nitpick: I'd put this new parameter after `pre_project` and `post_project` to match the ordering of the form.

Reply:
Done